### DISCUSSION TITLE
Port of Crazyflie 2.1 firmware to other platforms

### DISCUSSION THREAD

**Discussion by bzz1337** [bitcraze/discussions#1106] [Q&A]
I couldn't find much info about this. Are there any examples of people porting the Crazyflie firmware to other (preferably F7-based) platforms? If not, do you think it is feasible/hard to port it to some popular flight controllers?
I really like what you are doing, especially the lighthouse board, but already have some other hardware platforms that I would like to use in your ecosystem

**Comment by knmcguire** [comment]
Currently from within Bitcraze, we are not working on porting the firmware to a F7 platform (yet). 

Perhaps something interesting for you is that we are working on something that will make it easier for non-crazyflie platforms to make use of the lighthouse system. I don't want to reveal too much of that yet but we should have a [blogpost](https://www.bitcraze.io/blog/) about that soon, so keep an eye on it!
